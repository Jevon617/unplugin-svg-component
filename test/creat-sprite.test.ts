import path from 'path'
import { expect, test } from 'vitest'
import createSvgSprite from '../src/core/sprite'

test('createSprite', async () => {
  const { symbolIds } = await createSvgSprite({
    iconDir: path.resolve(__dirname, './icons'),
  })
  expect(symbolIds).matchSnapshot()
})
