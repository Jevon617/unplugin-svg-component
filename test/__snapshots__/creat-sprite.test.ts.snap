// Vitest Snapshot v1

exports[`createSprite 1`] = `
Set {
  "icon-addUser",
  "icon-card2",
  "logo",
  "common-icon-add",
  "common-icon-add3",
  "common-icon-addUser",
  "common-icon-addUsers",
  "common-icon-addx",
  "common-icon-apple",
  "common-icon-banner",
  "common-logo",
}
`;
